src/pages/Query.tsx

```diff
@@ -423,9 +410,7 @@ const ExplanationButton = () => {
               <ol className="list-decimal pl-5">
                 <li>
                   Enter the URL of the GitHub repository you want to query. You
-                  also need to enter your GitHub token to be able to query the
-                  database. This token will be used to download the repository.
-                  Only read access is required.
+                  can also specify the branch.
```

Comment:
Let's make it:

"Enter the URL of the GitHub repository you want to query. Optionally, you may specify the branch of the repository to query. By default, the main branch is queried."